h/h_api/bulk_api/entry_point.py

```diff
@@ -130,8 +132,9 @@ def _bytes_to_lines(stream, chunk_size=16384):
             while b"\n" in block:
                 head, block = block.split(b"\n", 1)
 
+                line += head
                 if line:
-                    yield line + head
+                    yield line
```

Comment:
This was flat wrong and only yielded the last item with short lines. I've no idea why this was passing the tests which assert we had 4 items. Might be a difference between long and short lines. I really don't know.